packages/elements/package.json

```diff
@@ -26,20 +26,21 @@
   },
   "customExports": {
     ".": {
-      "default": "./src/index.js"
+      "default": "./src/index.ts"
     }
   },
   "main": "dist/index.js",
   "module": "dist-module/index.js",
```

Comment:
```suggestion
  "main": "dist/index.ts",
  "module": "dist-module/index.ts",
```

Reply:
No that‘s incorrect, these build directories only contain JS files